lib/merlee.js

```diff
@@ -61,6 +56,32 @@ class Merlee {
     this.server.on('request', (req, res) => {
       // error cannot set headers after they have been sent
       _static(req, res, this.options.static);
+
+      // go through all the middleware and check if the "next" function has been called in the last 
+      // middleware
+
+      let cmw = 0; // current middleware
+      const midw = this.options.middleware;
+      let calledNextAtEnd = false;
+      
+      const next = () => {
+        if (cmw < midw.length - 1) {
+          cmw++;
+          midw[cmw](req, res, next);
+        } else if (cmw == midw.length - 1) {
+          calledNextAtEnd = true;
+        }
+      }
+
+      if (midw) {
+        if (midw.length > 0) {
```

Comment:
```suggestion
        if (midw && midw?.length > 0) {
```
if know you know what i mean. the `?` should only run when the class passed has a length method'ish i know you know what i mean though lol

Reply:
Oh yeah makes sense